lib/install/validate/config_to_data.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
this file convert VirtualContainerHostConfigSpec to Data structure, and then another file converter.go will convert Data structure back to command option, these two functions are covered by test cases in converter_test.go, so I didn't write any test case for this file, because the verification of string map is a lot more straight forward than a structure.